plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -303,13 +422,17 @@ protected boolean canHandle(Network network, Service service) {
         }
 
         if (!_networkModel.isProviderForNetwork(getProvider(), network.getId())) {
-            s_logger.debug("NuageElement is not a provider for network " + network.getDisplayText());
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("NuageElement is not a provider for network " + network.getDisplayText());
```

Comment:
Please provide identifying/context information of Nuage element to assist debugging.